### ISSUE TITLE
[1.7.0] Startup crash on Arch Linux

### ISSUE THREAD

**Issue by JohanLorenzo** [atom/atom#11444]
### Prerequisites

* [x] Can you reproduce the problem in [safe mode](http://flight-manual.atom.io/hacking-atom/sections/debugging/#check-if-the-problem-shows-up-in-safe-mode)?
* [x] Are you running the [latest version of Atom](http://flight-manual.atom.io/hacking-atom/sections/debugging/#update-to-the-latest-version)?
* [x] Did you check the [debugging guide](http://flight-manual.atom.io/hacking-atom/sections/debugging/)?
* [x] Did you check the [FAQs on Discuss](https://discuss.atom.io/c/faq)?
* [x] Are you reporting to the [correct repository](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#atom-and-packages)?
* [x] Did you [perform a cursory search](https://github.com/issues?q=is%3Aissue+user%3Aatom+-repo%3Aatom%2Felectron) to see if your bug or enhancement is already reported?

For more information on how to write a good [bug report](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#how-do-i-submit-a-good-bug-report) or [enhancement request](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#how-do-i-submit-a-good-enhancement-suggestion), see the `CONTRIBUTING` guide.

### Description

Startup crash on Arch Linux.

### Steps to Reproduce

1. Upgrade atom-editor (1.6.2-1 -> 1.7.0-1) thanks to the AUR.
1. `atom --safe --clear-window-state --foreground`. Regular `atom .` from whatever directory also fails.

**Actual behavior:**
![screenshot from 2016-04-13 11-54-36](https://cloud.githubusercontent.com/assets/5907366/14489674/d12d23a4-016e-11e6-9062-8590a2fe4940.png)

```
[11167:0413/115113:ERROR:browser_main_loop.cc(203)] Running without the SUID sandbox! See https://code.google.com/p/chromium/wiki/LinuxSUIDSandboxDevelopment for more information on developing with the sandbox on.
[11167:0413/115116:WARNING:accelerator_util.cc(118)] Invalid accelerator token: command
[11167:0413/115116:WARNING:accelerator_util.cc(118)] Invalid accelerator token: command
[11167:0413/115116:WARNING:accelerator_util.cc(118)] Invalid accelerator token: command
[11167:0413/115116:WARNING:accelerator_util.cc(118)] Invalid accelerator token: command
App load time: 1910ms
[11167:0413/115117:INFO:CONSOLE(52)] "Window load time: 1199ms", source: file:///usr/share/atom/resources/app.asar/static/index.js (52)
[11167:0413/115118:INFO:CONSOLE(0)] "/deep/ combinator is deprecated. See https://www.chromestatus.com/features/6750456638341120 for more details.", source:  (0)
Failed to get crash dump id.
Report Id: 
```

### Versions
OS: Arch Linux, with the most recent packages.

```
atom --version
Atom    : 1.7.0
Electron: 0.36.8
Chrome  : 47.0.2526.110
Node    : 5.1.1
```

```
apm --version 
apm  1.9.2
npm  2.13.3
node 0.10.40
python 3.5.1
git 2.8.0
```

**Comment by Nikki1993** [comment]
This already has been reported and requires update to Electron component in order to fix. See relevant issues here https://github.com/atom/atom/pull/11348 
https://github.com/atom/atom/pull/10983

**Comment by Dundee** [comment]
Quick fix:

sed -i -e "s/\"electronVersion\"\:\ \"0\.36\.8\"/\"electronVersion\"\:\ \"0\.36\.12\"/" package.json

https://aur.archlinux.org/packages/atom-editor

**Comment by marcopompili** [comment]
@Dundee 

> sed -i -e "s/\"electronVersion\":\ \"0.36.8\"/\"electronVersion\":\ \"0.36.12\"/" package.json

This line goes into the prepare function of the PKGBUILD, right ?

**Comment by conraid** [comment]
@Dundee 

In Slackware -current 64bit this fix the crash. Thanks

**Comment by Dundee** [comment]
@marcopompili Yep

**Comment by 50Wliu** [comment]
Closing in favor of #11198.

**Comment by JohanLorenzo** [comment]
Fixed in AUR by jugs with 1.7.0-2. Thank you all for the details :)
